src/types.ts

```diff
@@ -225,7 +231,8 @@ export interface HomeAssistant {
   callService: (
     domain: string,
     service: string,
-    serviceData?: { [key: string]: any }
+    serviceData?: { [key: string]: any },
+    target?: { [key: string]: any }
```

Comment:
Actually the `types.ts` typehints a `HassServiceTarget` as stated above.
if you want you can just add the https://github.com/home-assistant/frontend/blob/95c6adc739a1c632b36f97719e06f5e1f609da37/src/types.ts#L183 
To your PR

Reply:
Thank you, I have added it.